### DISCUSSION TITLE
archives/741/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#15] [Announcements]
# archives/741/

偶然逛 ArchWiki 的 微信 页面，发现在 Arch 下提供了很多种运行微信的方案，在我的 ARM 版 ARCH 下搜索了一下，发现提供了一个基于 electron 直接可用的微信： $ sudo pacman -Ss wechat archlinuxcn/electronic-wechat-uos-bin v2.3.1.fix-2 Linux 下更

https://www.frytea.com/archives/741/

**Comment by faonecze** [comment]
自从manjaro到arch后就没找到合适的wechat方法。。。用来用去换到win11了。。。。

**Reply by Freed-Wu** [reply]
我用的 <https://aur.archlinux.org/packages/wechat-uos>。除了聊天就没有别的功能了。
